frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js

```diff
@@ -23,11 +23,14 @@ function GameStatistics({ finalScores, setIsStatisticsDisplayed, handleResetGame
         <div className="table-item">score</div>
         <div className="table-item">{finalScores.white}</div>
       </div>
+
       <Button
         className="btn--primary new-game-btn"
         onClick={() => {
           setIsStatisticsDisplayed(false);
           handleResetGame();
+          stopwatchStart();
+          resetGoalsArray();
```

Comment:
It would be better to wrap those 4 callbacks into one function and pass it as a single prop (e.g. `onNewGameRequested`).

Reply:
Done